nexteuropa_user_management.install

```diff
@@ -88,11 +91,70 @@ function nexteuropa_user_management_install() {
     $user_1_wrapper->roles->set($user_1_roles);
     $user_1_wrapper->save();
   }
+
+  // To not duplicate code, call the update hook directly.
+  // @codingStandardsIgnoreLine
+  nexteuropa_user_management_update_7001();
```

Comment:
Please do not call an hook_update inside the hook_install.
As alternative I propose to move the code inside the function nexteuropa_user_management_update_7001 to an helper function in a .inc file.
This helper function can be then called on the hook_install and as well on the hook_update afterwards.

Reply:
fixed